client/jetpack-cloud/components/sidebar/header/profile-dropdown.tsx

```diff
@@ -0,0 +1,95 @@
+import { Gravatar } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useRef, useState } from 'react';
+import { useDispatch, useSelector } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
+import { redirectToLogout } from 'calypso/state/current-user/actions';
+import { getCurrentUser } from 'calypso/state/current-user/selectors';
+import useOutsideClickCallback from './use-outside-click-callback';
+import './style.scss';
+
+type DropdownMenuProps = {
+	isExpanded: boolean;
+};
+const DropdownMenu = ( { isExpanded }: DropdownMenuProps ) => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const onGetHelp = useCallback( () => {
+		dispatch( recordTracksEvent( 'calypso_jetpack_sidebar_gethelp' ) );
+	}, [ dispatch ] );
+	const onSignOut = useCallback( () => {
+		dispatch( recordTracksEvent( 'calypso_jetpack_sidebar_signout' ) );
+		dispatch( redirectToLogout() );
+	}, [ dispatch ] );
+
+	return (
+		<ul className="jetpack-cloud-sidebar__profile-dropdown-menu" hidden={ ! isExpanded }>
+			<li className="jetpack-cloud-sidebar__profile-dropdown-menu-item">
+				<a
```

Comment:
Can we use the `Button` from `import { Button } from '@automattic/components';` and set the href along with the `borderless` prop?